Improved code styles

drakeet opened the pull request:
I am very pleased to see this project. However, I found some minor problems, and made corrections, hope that it will get better and better, to help more people find out the Agera.

The following are the updates:
- Changed `R.id.xxx_tv` to `R.id.xxx` and the field variable names
  associated with these IDs
- Fixed some hump naming problems such as `mValue1TVupdatable` to
  `mValue1Updatable`
- Reformated the XML layout file by
  [LayoutFormatter](https://github.com/drakeet/LayoutFormatter)
- Changed gradle version to `2.1.0` because the `2.2.0-alpha4` need
  Android Studio 2.2 preview 4

drakeet committed:
Improved code styles

- Changed `R.id.xxx_tv` to `R.id.xxx` and the field variable names
associated with these IDs
- Fixed some hump naming problems such as `mValue1TVupdatable` to
`mValue1Updatable`, `UiUtils` to `UIUtils`
- Reformated the XML layout file by
[LayoutFormatter](https://github.com/drakeet/LayoutFormatter)
- Changed gradle version to v2.1.0 because the v2.2.0-alpha4 need
Android Studio 2.2 preview 4

JoseAlcerreca commented on app/src/main/java/com/example/android/codelabs/agera/UiUtils.java:
```diff
@@ -21,18 +21,18 @@
 import android.view.animation.TranslateAnimation;
 
-public class UiUtils {
+public class UIUtils {
```
This is incorrect.

drakeet replied:
UI is the abbreviation of Interface User, should use all capital.

drakeet replied:
I checked the document, this point you're right, for readable, it's reasonable and acceptable:

![image](https://cloud.githubusercontent.com/assets/5214214/16766082/66540e1a-486a-11e6-9374-5a2a903dfbd3.png)

https://source.android.com/source/code-style.html

JoseAlcerreca commented on app/src/androidTest/java/com/example/android/codelabs/agera/step2/Step2ActivityFinalTest.java:
```diff
@@ -44,22 +44,22 @@
-        onView(withId(R.id.step2_value_tv)).check(matches(withText("0")));
-        onView(withId(R.id.increment_bt)).perform(click());
-        onView(withId(R.id.step2_value_tv)).check(matches(withText("1")));
+        onView(withId(R.id.step2_value)).check(matches(withText("0")));
```
why? IDs are not typed.

drakeet replied:
There is no need to use type(such as `tv`, `bt`) in IDs, and in the entire source code of Android SDK is not seen so named. :P

JoseAlcerreca replied:
Of course there's no need, but AFAIK there's no rule that says they shouldn't. :)

JoseAlcerreca commented on app/src/main/java/com/example/android/codelabs/agera/step2/Step2Activity.java:
```diff
@@ -31,23 +31,23 @@
     private Updatable mTextValueUpdatable;
 
-    private TextView mValueTv;
+    private TextView mValueTextView;
```
No strong feelings about this, but ok.

JoseAlcerreca commented on build.gradle:
```diff
@@ -5,9 +5,9 @@ buildscript {
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0-alpha4'
+        classpath 'com.android.tools.build:gradle:2.1.0'
```
Please don't change this.

drakeet replied:
Sorry, but if someone is not using Android Studio 2.2 preview 4, he can not use this version of the gradle, by the way, the latest version of the AS is 2.2 Preview 5 with `2.2.0-alpha5` gradle and cannot use alpha4
![image](https://cloud.githubusercontent.com/assets/5214214/16765377/3e21fe7e-4866-11e6-87b1-6b6eee9036a6.png)

JoseAlcerreca commented:
Thanks drakeet for the PR.

However, most of the changes are nits (and some can't be accepted, I've commented on some). I only see a couple of bigger issues like fields not being prefixed with the 'm'. 

Since the codelab would have to be updated with these changes, I'm not going to merge it yet.

drakeet commented:
Thank you for your reply ^ ^ and the reason I pull this request is:

I think, Google's code should be everyone's norms, is a mirror. So when I see minor problem and want to improve it. :P

Commit:
Reset UIUtils to UiUtils

 According to the
[android-code-style](https://source.android.com/source/code-style.html)
for readable, it's reasonable and acceptable
